apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -77,17 +79,17 @@ export class IntegrationsController {
   @Get('/active')
   @ApiOkResponse({
     type: IntegrationResponseDto,
-    description: 'The list of active integrations belonging to the organization that are successfully returned.',
+    description: 'The list of active integrations belonging to the environment that are successfully returned.',
```

Comment:
I was expecting that this endpoint had to be based on organization and not in environment. Maybe we should clarify that because API can be confusing now as the generic GET /integrations endpoint is based on organization thus the breaking change.

Reply:
100% agree, did it like this because in the DOD the task was written it should be on the environment level. [REF](https://linear.app/novu/issue/NV-2420/update-the-active-integrations-endpoint)

Reply:
refactored to the organization level 🫣